byzcoin/bcadmin/main.go

```diff
@@ -472,7 +472,7 @@ var cmds = cli.Commands{
 								Usage:  "the ByzCoin config to use (required)",
 							},
 							cli.StringFlag{
-								Name:  "instID",
+								Name:  "instid",
```

Comment:
Please use `"instid, i"` in order to allow the short argument -i for the frequently used instance id. See https://github.com/urfave/cli#alternate-names

Reply:
ok, done